README.md

```diff
@@ -1,11 +1,31 @@
 # STAC ipyleaflet
 
+### What
+stac_ipyleaflet is a customized version of [ipyleaflet](https://ipyleaflet.readthedocs.io/en/latest/) built to be an in jupyter-notebook interactive mapping library that can access STAC catalogs (currently only the MAAP STAC) to help better visualize cloud-optimized geotiffs. The library provides widgets that is meant to help users working in Jupyter Notebook environments to quickly visualize and interact with geospatial data at different stages throughout their process.
+
+### Why
+While the intended users of this library are technical and can write code on their own to plot and analyze data, it's very time consuming to export results and visualize/explore data in & out of a jupyter notebook. So this library helps users visualize remotely sensed data as quickly as possible at scale as fast as possible.
+
+![stac_ipyleaflet as part of the explore/visualization solution](/public/images/about-map-visualization-solution.png)
+> stac_ipyleaflet as part of the explore/visualization solution
+
```

Comment:
📓 : This was gathered from the resources listed in this [slack thread](https://developmentseed.slack.com/archives/CAE4M5VGS/p1689085082053879) and much of this is copy and pasted. Please provide feedback if anything is mis-worded, missing or if more context can be provided.

Reply:
Maybe we can add more on who are the intended users, and also a comment about how overlaps/not with more common tools like QGIS.

----

The intended users of this library are members of the scientific community who are doing research in areas like climate and active remote sensing. While they can write code to visualize and analyze data, they find it time consuming and may end up exporting results to visualize/explore data in & out of a jupyter notebook. This library allows them to visualize remotely sensed data as quickly as possible at scale as fast as possible right in the notebook. It does not replace full featured GIS tools but offers enough visualization capabilities to support their scientific analysis cycles.

Reply:
added